vehicle/simvehicle.cc

```diff
@@ -1798,6 +1814,56 @@ vehicle_t::~vehicle_t()
 }
 
 
+ribi_t::ribi vehicle_t::get_direction_of_travel() const
+{
+	ribi_t::ribi dir = get_direction();
+	if(reversed)
+	{
+		switch(dir)
```

Comment:
Probably `ribi_t::backward` is enough.

https://github.com/teamhimeh/simutrans/blob/dfd4e028846e92c9a606b04353f5e0d31c237298/dataobj/ribi.h#L244

Reply:
TODO: consider this